src/main/java/htsjdk/samtools/cram/build/CRAMReferenceRegion.java

```diff
@@ -73,7 +74,7 @@ public CRAMReferenceRegion(final CRAMReferenceSource cramReferenceSource, final
                 final SAMSequenceRecord sequence = samFileHeader.getSequence(referenceIndex);
                 referenceBases = referenceSource.getReferenceBases(sequence, true);
                 if (referenceBases == null) {
-                    throw new IllegalArgumentException(
+                    throw new CRAMException(
```

Comment:
I'm not sure what the original intent of CRAMException was, but starting with the CRAM rewrite I restricted it's use to internal error conditions, such as malformed input as reported in https://github.com/broadinstitute/picard/issues/1624. Not all of the CRAM codebase is conforming yet, but I don't think this is a case where CRAMException makes sense, since this is a user error.